ts-motive-zone-app/src/Transaction/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react'
 import Chart from './Chart'
 import MonthYearPicker from './MonthYearPicker'
+import '../assets/styles/_transaction.scss'
```

Comment:
nên có một file all.scss gòm toàn bộ css lại thành một css duy nhất.
và import file css này trong ts-motive-zone-app/src/index.tsx

Reply:
Đã update ạ